### PR TITLE
Add ws bootnodes in polkadot, kusama, rococo and westend chainspecs

### PR DESCRIPTION
**Pull request by wirednkod**
Add bootnodes in relay chains polkadot, kusama, rococo and westend that are being used from substrate-connect and smoldot light client.

### CODE REVIEW

File: Cargo.lock
Comment (bkchr) [inline Cargo.lock:453]: Could you please revert this.

**Comment by tomaka** [comment]
Can we not merge this? Is anything blocking?

**Comment by bkchr** [comment]
> Can we not merge this? Is anything blocking?

From my side nothing. I just don't wanted to approve because IDK if these are the correct addresses etc and thus, I requested a review from @lovelaced.

**Review by lovelaced — APPROVED** [review]

**Pull request merged**
